package.json

```diff
@@ -1,64 +1,25 @@
 {
   "name": "irving",
-  "version": "1.0.0",
+  "private": true,
```

Comment:
Necessary to prevent issues with publishing root package (which should never be published anyway)